packages/hs-react-ui/src/components/Appendix/Appendix.stories.tsx

```diff
@@ -1,20 +1,35 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 
-const design = {
-  type: 'figma',
-  url: 'https://www.figma.com/file/3r2G00brulOwr9j7F6JF59/Generic-UI-Style?node-id=0%3A1',
-};
+import styled from 'styled-components';
+import { Div } from 'src/htmlElements';
 
-storiesOf('Appendix', module).add(
-  'Appendix',
-  () => {
-    return (
-      <div>
-        <h1>'Welcome to the HS-UI Appendix'</h1>
-        {/* <img src={require('/ThemingExample.png')} alt="Theming Example" /> */}
-      </div>
-    );
-  },
-  { design },
-);
+const DocContainer = styled(Div)`
+  justify-self: flex-start;
+  padding: 2rem;
+  max-width: 60rem;
+`;
+const FullWidthImage = styled.img`
+  width: 100%;
+`;
+
+// TODO: Turn these images into html
```

Comment:
What exactly does this comment mean?

Reply:
The images there - they should be not images since they're basically images of text - we can do text using our web development experience and skills :)

Reply:
Roger